di_website/spotlight/models.py

```diff
@@ -18,12 +18,17 @@ class Meta():
     country_code = models.CharField(max_length=100, help_text='e.g. UG, KE', default='')
     country_name = models.CharField(max_length=255)
     currency_code = models.CharField(max_length=100, help_text='UGX, KES', default='')
-
+    datasources_description = models.TextField(help_text='A description for data sources section', default='')
+    datasources_url_title = models.TextField(help_text='A title for data sources url link', default='')
```

Comment:
Caption? ... title sounds like something above the actual URL :)

Reply:
Yea, caption sounds about right, I updated this as well